### PR TITLE
Restore SonarCloud configuration

### PR DESCRIPTION
**Pull request by Sondergaard**
## Description

This PR will restore SonarCloud workflow in the build pipeline. The reason for this is that we need to be ready for a security audit in near future.

At a later stage this will be moved to be a part of the generic build flow in geh-core.

### CODE REVIEW

File: .github/workflows/sonarcloud-dotnet.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
+name: SonarCloudDotnet
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - source/coordinator/**
+      - source/messaging/**
```
Comment (ZavezX) [inline .github/workflows/sonarcloud-dotnet.yml:8]: Do we need to test messaging? I believe that the messaging is moved to external repo and referenced via nuget where messaging is used.

**Comment by ZavezX** [comment]
I think we had sonar cloud on python part too, is that also meant to be restored as well as the .NET part?

**Comment by Sondergaard** [comment]
> I think we had sonar cloud on python part too, is that also meant to be restored as well as the .NET part?

The config for sonarcloud python is still present in the repo - no modifications needed for this part

File: .github/workflows/sonarcloud-dotnet.yml (hunk 1 above)
Comment (Sondergaard) [reply .github/workflows/sonarcloud-dotnet.yml:8]: The path still exist in the main branch. This is just a restore of the previous configuration. 

I hope that you can verify if it's still correct.

Comment (jonasdmoeller) [reply .github/workflows/sonarcloud-dotnet.yml:8]: I will remove path in main branch in another PR.

**Review by jonasdmoeller — APPROVED** [review]

**Review by ZavezX — APPROVED** [review]

File: .github/workflows/sonarcloud-dotnet.yml (hunk 1 above)
Comment (jonasdmoeller) [reply .github/workflows/sonarcloud-dotnet.yml:8]: I just realised the old messaging code is still used in our Coordinator solution, and hence cannot be removed yet.

**Pull request merged**